scripts/quail_maker_web.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import csv
+from datetime import datetime, timedelta, timezone
+from flask import Flask, send_file
+from glob import glob
+from tempfile import NamedTemporaryFile
+
+
+def get_csv_files(directory):
+    path = f"{directory}/*.csv"
+    files = glob(path)
+    return sorted(files)
+
+
+def gather_data(directory="./"):
+    files = get_csv_files(directory)
+
+    data = []
+
+    edt = timezone(timedelta(hours=-4))
```

Comment:
Hardcoded for EDT for now. Would be better to get this from a library.

Reply:
Yes